### ISSUE TITLE
ezSQL contants redefined

### ISSUE THREAD

**Issue by abelcallejo** [ezSQL/ezSQL#104]
In any case that the ezSQL constants have a conflict or the library got included twice, ezSQL throws some PHP notices like so:
```
Notice: Constant EZSQL_VERSION already defined in [...]/ezSQL/shared/ez_sql_core.php on line 17
Notice: Constant object already defined in [...]/ezSQL/shared/ez_sql_core.php on line 18
```

**Comment by jv2222** [comment]
Hmm. Why would we merge that? Seems like that's exactly how it _SHOULD_ behave!

**Comment by gnowxilef** [comment]
oh, now that i look at the issue again, good point. where are you getting this conflict/including the library twice?

**Comment by abelcallejo** [comment]
@gnowxilef I used ezSQL as a library inside a **WordPress** theme. I included ezSQL and called it. Later on during runtime, WordPress also called ezSQL for its[ **wpdb** class](https://codex.wordpress.org/Class_Reference/wpdb). Using WordPress wpdb solely is not an option for that scenario because ezSQL was used to connect to PostgreSQL while WordPress is using ezSQL for MySQL.

Have to be included twice but its throwing the said PHP notices.
